pkg/controller/registrysyncer/registrysyncer.go

```diff
@@ -312,20 +315,24 @@ func testInputImageStreamTagFilterFactory(
 	imageStreams sets.String,
 	imageStreamPrefixes sets.String,
 	imageStreamNamespaces sets.String,
+	deniedImageStreams sets.String,
 ) objectFilter {
 	l = logrus.WithField("subcomponent", "test-input-image-stream-tag-filter")
 	return func(nn types.NamespacedName) bool {
+		imageStreamName, err := imageStreamNameFromImageStreamTagName(nn)
```

Comment:
this returns `namespace/name` ?

Reply:
It returns `types.NamespacedName`. Then we compare with `imageStreamName.String()` which is form of `namespace/name`.